examples/display/sdkconfig.bsp.m5stack_core_2

```diff
@@ -25,3 +25,6 @@ CONFIG_LV_USE_PERF_MONITOR=y
 # CONFIG_LV_MEM_CUSTOM=y
 # CONFIG_LV_MEMCPY_MEMSET_STD=y
 # CONFIG_LV_SPRINTF_CUSTOM=y
+
+CONFIG_LV_USE_FLOAT=y
+CONFIG_LV_USE_GESTURE_RECOGNITION=y
```

Comment:
We shouldn't add it for core_2 (it is not working properly with multitouch, right?)